src/rust/ensogl/lib/core/src/display/scene.rs

```diff
@@ -857,7 +857,7 @@ impl SceneData {
         let keyboard        = Keyboard::new();
         let network         = &frp.network;
         let extensions      = Extensions::default();
-        let bg_color_var    = style_sheet.var("application.background.color");
+        let bg_color_var    = style_sheet.var("application . background . color");
```

Comment:
As we talked on DM, this here coouldn't be changed because of cyclic imports problem, as theme library uses core lib, so core cannot use theme anymore. This is the only place.